jaide/core.py

```diff
@@ -250,12 +252,10 @@ def commit(self, commands="", confirmed=None, comment=None,
         # passed, use 'annotate system' to make a blank commit
         if not commands:
             commands = 'annotate system ""'
-        clean_cmds = []
-        for cmd in clean_lines(commands):
-            clean_cmds.append(cmd)
+        clean_cmds = [x for x in clean_lines(commands)]
         # try to lock the candidate config so we can make changes.
         self.lock()
-        self._session.load_configuration(action='set', config=commands)
+        self._session.load_configuration(action=action, config=clean_cmds, format="text")
```

Comment:
Will adding the `format='text'` kwarg break other action types (set in particular?) I'm assuming the other option for format is xml?